### PR TITLE
initial port of ts simple http api to py

### PR DESCRIPTION
**Pull request by hiteshboinpally**
Ported the [TypeScript example](https://github.com/pulumi/examples/tree/master/aws-ts-apigatewayv2-http-api-quickcreate) of the same name to Python.

- Note: seems like the `pulumi/examples` README's "API Gateway HTTP API quickstart" link in the TypeScript table under AWS is broken. Should link to the example I ported, but instead goes to a 404. 
    - Current link is [https://github.com/pulumi/examples/blob/master/aws-ts-apigatewayv2-http-api-quick**start**](https://github.com/pulumi/examples/blob/master/aws-ts-apigatewayv2-http-api-quickstart), most likely should be [.../aws-ts-apigatewayv2-http-api-quick**create**](https://github.com/pulumi/examples/blob/master/aws-ts-apigatewayv2-http-api-quickcreate) instead. 
    - Named my folder based on the example folder, rather than broken link, but can easily change depending on which is preferred
- Slightly edited README compared to TS example. Added pre-reqs and a little more info about what the API does (executes simple lambda function).
- Set copyright date until 2021 because I saw other examples with 2016-2021

### CODE REVIEW

File: aws-py-apigatewayv2-http-api-quickcreate/README.md [hunk 1]
```diff
@@ -0,0 +1,81 @@
+[![Deploy](https://get.pulumi.com/new/button.svg)](https://app.pulumi.com/new)
+
+# AWS API Gateway V2 HTTP API Quickstart
+
+Set up a simple HTTP API using AWS API Gateway V2. The API executes a simple Lambda with function
```
Comment (komalali) [inline aws-py-apigatewayv2-http-api-quickcreate/README.md:5]:
```suggestion
Set up a simple HTTP API using AWS API Gateway V2. The API executes a simple Lambda function 
```

File: aws-py-apigatewayv2-http-api-quickcreate/__main__.py [hunk 2]
```diff
@@ -0,0 +1,48 @@
...
+    function=lambda_function)
+
+# Set up the API Gateway
+apigw = aws.apigatewayv2.Api("httpApiGateway", 
+    protocol_type="HTTP",
+    route_key="GET /",
+    target=lambda_function.invoke_arn)
+
+# Export the API endpoint for easy access
+pulumi.export("endpoint", apigw.api_endpoint)
```
Comment (komalali) [inline aws-py-apigatewayv2-http-api-quickcreate/__main__.py:48]: Missing a newline

```suggestion
pulumi.export("endpoint", apigw.api_endpoint)

```

File: aws-py-apigatewayv2-http-api-quickcreate/app/index.js [hunk 3]
```diff
@@ -0,0 +1,7 @@
+exports.handler =  async function(event, context) {
+    console.log("EVENT: \n" + JSON.stringify(event, null, 2))
+    return {
+        statusCode: 200,
+        body: "Hello, Pulumi!"
+    };
+  }
```
Comment (komalali) [inline aws-py-apigatewayv2-http-api-quickcreate/app/index.js:7]: Missing a newline
```suggestion
  }

```

**Review by komalali — COMMENTED** [review]
Looks great, just some minor aesthetic comments :) 

@hiteshboinpally would you mind fixing the broken link in the readme table for nodejs and also adding a line for the python equivalent to the main README?

**Comment by hiteshboinpally** [comment]
> Looks great, just some minor aesthetic comments :)
> 
> @hiteshboinpally would you mind fixing the broken link in the readme table for nodejs and also adding a line for the python equivalent to the main README?

Yep, sure thing! I'll make the aesthetic changes and then edit the main README.

**Review by komalali — APPROVED** [review]

**Pull request merged**